airflow/operators/dagrun_operator.py

```diff
@@ -52,26 +59,22 @@ def __init__(
             self,
             trigger_dag_id,
             python_callable=None,
+            execution_date=None,
             *args, **kwargs):
         super(TriggerDagRunOperator, self).__init__(*args, **kwargs)
         self.python_callable = python_callable
         self.trigger_dag_id = trigger_dag_id
+        self.execution_date = execution_date
 
     def execute(self, context):
         dro = DagRunOrder(run_id='trig__' + timezone.utcnow().isoformat())
         if self.python_callable is not None:
             dro = self.python_callable(context, dro)
         if dro:
-            with create_session() as session:
```

Comment:
So the session is no longer needed with the changes in `airflow/api/common/experimental/trigger_dag.py` ?